src/index.js

```diff
@@ -64,6 +66,25 @@ const defaultNodeRenderers = {
     defaultInline(INLINES.ENTRY_HYPERLINK, node, key),
   [INLINES.EMBEDDED_ENTRY]: (node, key) =>
     defaultInline(INLINES.EMBEDDED_ENTRY, node, key),
+  [BLOCKS.EMBEDDED_ASSET]: (node, key, next) => {
+    const attrs = {
+      title: node.data.target.fields.title,
+      description: node.data.target.fields.description,
+      url: node.data.target.fields.file.url,
+      fileName: node.data.target.fields.file.fileName,
+      contentType: node.data.target.fields.file.contentType,
+    };
+
+    if (attrs.contentType.startsWith("image/")) {
+      return h("img", {
+        key,
+        src: attrs.url,
+        alt: attrs.title,
+        title: attrs.title,
+        name: attrs.fileName
+      }, next(node.content, key, next));
+    }
+  },
```

Comment:
Could you please split this to it's own PR 👍. I have some reservations about merging opinionated stuff as default.